Cargo.toml

```diff
@@ -19,17 +19,33 @@ fnv = "1.0.7"
 futures = "0.3.19"
 lazy_static = "1.4.0"
 libipld = { version = "0.15.0", default-features = false }
-libp2p = { version = "0.50.0", features = ["request-response"] }
+libp2p = { version = "0.52", features = ["request-response"] }
 prometheus = "0.13.0"
-prost = { version = "0.11", optional = true }
 thiserror = "1.0.30"
 tracing = "0.1.29"
 unsigned-varint = { version = "0.7.1", features = ["futures", "std"] }
 
+quick-protobuf = { version = "0.8.1", optional = true }
+
 [dev-dependencies]
 async-std = { version = "1.10.0", features = ["attributes"] }
 env_logger = "0.9.0"
-libipld = { version = "0.15.0", default-features = false, features = ["dag-cbor"] }
-libp2p = { version = "0.50.0", features = ["tcp", "noise", "yamux", "rsa", "async-std"] }
-multihash = { version = "0.17.0", default-features = false, features = ["blake3", "sha2"] }
-tracing-subscriber = { version = "0.3.5", features = ["env-filter", "tracing-log"] }
+libipld = { version = "0.15.0", default-features = false, features = [
+    "dag-cbor",
+] }
+libp2p = { version = "0.52", features = [
+    "tcp",
+    "noise",
+    "yamux",
+    "rsa",
+    "ed25519",
+    "async-std",
+] }
+multihash = { version = "0.17.0", default-features = false, features = [
```

Comment:
@mxinden to my knowledge, multihash in libp2p has been updated to 0.19, so I would assume it would be safe to update this as well? Havent checked the changes upstream so i would assume another crate would need to be imported

CC @rkuhn

Reply:
Correct. rust-libp2p v0.52.0 uses rust-multihash v0.19.0.

https://github.com/libp2p/rust-libp2p/blob/ce9821154a3bde53e38e72c511acbacb721573ce/Cargo.lock#L3333-L3335

> so I would assume it would be safe to update this as well? 

I don't see any reason not to do so. :+1: